french-soup/styles/style.css

```diff
@@ -7,3 +7,44 @@ html {
   font-family: 'Open Sans', sans-serif;
```

Comment:
Same comment here... nice work, however, you've placed this solution in the wrong folder. It should be in ```apple-pie``` and not ```french-soup```.